src/js/constants.js

```diff
@@ -29,6 +29,13 @@ var exports = {
   USER_BLOCK: "user_block",
   USER_COOKIE_BLOCK: "user_cookieblock",
 
+  BLOCKED_ACTIONS: [
```

Comment:
Could you make it into an object made up of existing constants? Something like:
```javascript
constants.BLOCKED = {
  USER_BLOCK: true,
};
```
Checking then a `BLOCKED.hasOwnProperty()` call, avoiding array scanning.